### DISCUSSION TITLE
【VPS-Ubuntu】+【alist 115 302重定向】+【davfs2挂载本地】+【plex】播放网络问题

### DISCUSSION THREAD

**Discussion by reecelu** [AlistGo/alist#7495] [General]
甲骨文VPS，挂载本地的文件夹已经修改了777全权限，davfs config也修改了 follow_redirect 1。试过文件可以从挂载文件夹拷贝到本地，但是plex播放就不行，每次提示网络错误。clouddrive2试过，cd2没有302重定向，可以播但是流量经过vps转发很慢。你们有遇到这个问题么？对了我的vps只有1C+1G内存，但配置低不会直接用不了吧，应该只是卡。

**Comment by Take2X** [comment]
你用甲骨文会经常出现115访问拦截的问题吗？新西经常出现，搞得我现在又本地托管了

**Reply by reecelu** [reply]
放弃oci了。。免费的东西才是最贵的，国内访问慢又不稳定，我现在是emby+alist+cd2+emby2alist部署到本地的nas了，然后买了个阿里云vps frp到本地的nas了，已经稳定运行两个月了，等vps涨价我再买个新的frp本地nas就好了，方便
